mne/io/fiff/raw.py

```diff
@@ -432,7 +433,7 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
                     break
 
     def fix_mag_coil_types(self):
-        """Fix Elekta magnetometer coil types
```

Comment:
oops, that's a leftover from my shuffling with docstrings. Reverting this change.